[2022-12-12 05:27:10,440] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_task_table manual__2022-12-12T05:27:08.938323+00:00 [queued]>
[2022-12-12 05:27:10,466] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_task_table manual__2022-12-12T05:27:08.938323+00:00 [queued]>
[2022-12-12 05:27:10,469] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 05:27:10,471] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 05:27:10,473] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 05:27:10,533] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_task_table> on 2022-12-12 05:27:08.938323+00:00
[2022-12-12 05:27:10,539] {standard_task_runner.py:52} INFO - Started process 7829 to run task
[2022-12-12 05:27:10,563] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'create_task_table', 'manual__2022-12-12T05:27:08.938323+00:00', '--job-id', '1661', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpn6djrc8g', '--error-file', '/tmp/tmpu34k3ts0']
[2022-12-12 05:27:10,570] {standard_task_runner.py:80} INFO - Job 1661: Subtask create_task_table
[2022-12-12 05:27:10,763] {task_command.py:369} INFO - Running <TaskInstance: first_dag.create_task_table manual__2022-12-12T05:27:08.938323+00:00 [running]> on host 7e39590a7807
[2022-12-12 05:27:11,421] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=create_task_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-12T05:27:08.938323+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-12T05:27:08.938323+00:00
[2022-12-12 05:27:11,521] {base.py:68} INFO - Using connection ID 'db' for task execution.
[2022-12-12 05:27:11,551] {dbapi.py:208} INFO - Running statement: 
                create table if not exists task (
                id SERIAL,
	            value_1 INT,
	            value_2 INT
            );
              , parameters: None
[2022-12-12 05:27:11,674] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=create_task_table, execution_date=20221212T052708, start_date=20221212T052710, end_date=20221212T052711
[2022-12-12 05:27:11,752] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 05:27:11,867] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
